### PR TITLE
Fix validation transforms in ResNet example

### PR DESCRIPTION
**Pull request by Yancey1989**
The transforms on the validation dataset are different from training data.
c.f. https://github.com/pytorch/examples/blob/master/imagenet/main.py#L225

### CODE REVIEW

**Review by shendiaomo — APPROVED** [review]

**Review by QiJune — APPROVED** [review]
LGTM

**Comment by shendiaomo** [comment]
> The transforms on the validation dataset are different from training data.
> c.f. https://github.com/pytorch/examples/blob/master/imagenet/main.py#L225

Great. This may lead to unstable accuracy on the validation dataset.
